### DISCUSSION TITLE
tutorials/intro-hotchocolate/10-arguments

### DISCUSSION THREAD

**Discussion by giscus** [apollographql-education/content-feedback#182] [course feedback]
# tutorials/intro-hotchocolate/10-arguments

Learn how to use GraphQL arguments and variables.

https://www.apollographql.com/tutorials/intro-hotchocolate/10-arguments

**Comment by scottrudy** [comment]
It's worth noting the SpotifyService will throw a `SpotifyWeb.ApiException<SpotifyWeb.Response>` if the id is not found and the GetPlaylistDetails entry point would need to handle that.
